android/tangram/src/com/mapzen/tangram/MapData.java

```diff
@@ -0,0 +1,100 @@
+package com.mapzen.tangram;
+
+public class MapData {
+
+    /**
+     * Construct a new MapData object for adding drawable data to the map
+     * @param name Name of the data source in the scene file for styling this object's data
+     */
+    public MapData(String name) {
+        this.name = name;
+        this.id = addDataSource(name);
+    }
+
+    /**
+     * Get the name of this data source
+     * @return The name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Remove all data from this source
+     * @return This object, for chaining
+     */
+    public MapData clear() {
+        clearSourceData(id);
+        return this;
+    }
+
+    /**
+     * Add geometry from a GeoJSON string to this data source
+     * @param data String of GeoJSON containing a Feature or FeatureCollection
+     * @return This object, for chaining
+     */
+    public MapData addGeoJSON(String data) {
+        addSourceGeoJSON(id, data);
+        return this;
+    }
+
+    /**
+     * Add a point geometry to this data source
+     * @param point Two-element array of longitude and latitude (in that order) in degrees
+     * @return This object, for chaining
+     */
+    public MapData addPoint(double[] point) {
+        addSourcePoint(id, point);
+        return this;
+    }
+
+    /**
+     * Add a line geometry to this data source
+     * @param line Array of points, where points are as described in {@link #addPoint(double[])}
+     * @return This object, for chaining
+     */
+    public MapData addLine(double[][] line) {
```

Comment:
True, since we're iterating over the coordinates anyway it would be simple to flip them if needed. However I feel conflicted about supporting LatLng ordering. We should be aiming for consistency across our data exchange interfaces and the GeoJSON spec is [pretty clear](http://geojson.org/geojson-spec.html#positions) on the ordering that coordinates should follow.